src/platform/python/_builder.py

```diff
@@ -1,19 +1,48 @@
 import cffi
+import distutils.ccompiler
+import json
 import os, os.path
+import re
 import shlex
 import subprocess
 import sys
 
+
+def split_cmake_list(value):
+    return list(filter(None,value.split(";")))
```

Comment:
```suggestion
    return [item for item in value.split(";") if item]
```
The existing approach is somewhat arcane (I had no idea what `filter(None, ...)` does), and looks copy-pasted from Stack Overflow.